tests/e2e/Adapter/Scopes/Relationships/OneToOneTests.php

```
@@ -2289,4 +2289,62 @@ public function testDeleteTwoWayRelationshipFromChild(): void
+        $database->createCollection($childCollection);
+        $database->createAttribute($parentCollection, 'name', Database::VAR_STRING, 255, true);
+        $database->createAttribute($childCollection, 'name', Database::VAR_STRING, 255, true);
+
+        $database->createRelationship(
+            collection: $parentCollection,
+            relatedCollection: $childCollection,
+            type: Database::RELATION_ONE_TO_ONE,
+            onDelete: Database::RELATION_MUTATE_RESTRICT
+        );
+
+        $parent = $database->createDocument($parentCollection, new Document([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$parent'. (Unused Code Rules)

(UnusedLocalVariable)